modules/caddyhttp/rewrite/caddyfile.go

```diff
@@ -25,20 +27,116 @@ func init() {
 
 // parseCaddyfile sets up the handler from Caddyfile tokens. Syntax:
 //
-//     rewrite [<matcher>] <to>
+//     rewrite [<matcher>] [<to>] {
+//         to                <string>
+//         method            <string>
+//         strip_path_prefix <string>
+//         strip_path_suffix <string>
```

Comment:
If "path" is obvious here, I wonder if we could omit that, to make this easier to write?